pyproject.toml

```diff
@@ -68,6 +68,8 @@ server = [
     "fastapi>=0.103.1",
     "pyyaml~=6.0",
     "optimade[mongo]",
+    "numpy>=1.20",
+    "jsonlines>=3.1",
```

Comment:
I think maybe these should still be optional server deps, or special `trajectories` deps?